.github/workflows/publish.yml

```diff
@@ -0,0 +1,55 @@
+# This github action builds Alibi Detect and publishes it to PyPI.
+# It is only run when a new tag is pushed to github.
+# See https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+name: Publish 📦 to PyPI 
+
+# Only run when tags starting with "v" are pushed
+on: 
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build-and-publish:
+    name: Build and publish
+    runs-on: ubuntu-18.04
+
+    steps:
+    # Setup version extractor (from tag)
+    - uses: nowsprinting/check-version-format-action@v3
+      id: version
+      with:
+        prefix: 'v'
+
+    # Setup Python
+    - uses: actions/checkout@master
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.9
+
+    # Build
+    - name: Install pypa/build
+      run: python -m pip install --user build==0.9.0
```

Comment:
Assume the specific version is just the latest stable one?

Reply:
Yep. I toyed with the idea of doing `build~=0.9` or something, but wondering if it is safer to pin to a specific version since we will be releasing builds straight off of it?